<launch>
	<node pkg="tello_driver" exec="tello_driver_main" name="tello_driver_main" namespace="tello" output="log">
    <!--<param name="drone_ip" value="127.0.0.1" />  for testing-->
  </node>
  <node pkg="image_view" exec="image_view" name="image_view" namespace="view" output="screen">
    <remap from="image" to="/processing/output_images"/>
  </node>
  <node pkg="tello_ros" exec="tello_simple_solution" name="auto_control" namespace="auto_control" output="log">
    <remap from="out_cmd_vel" to="/tello/cmd_vel"/>
    <remap from="srv_tello_action" to="/tello/tello_action"/>
    <remap from="action_response" to="/tello/tello_response"/>
  </node>
  <node pkg="tello_ros" exec="image_processing" name="image_processing" namespace="processing" output="log">
    <remap from="input_images" to="/tello/image_raw"/>
    <remap from="flight_status" to="/tello/flight_data"/>
  </node>
</launch>
